### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by auriee**
Address remaining comments #30

### CODE REVIEW

File: draft-ietf-pquip-pqc-engineers.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ The presence of a Cryptographically Relevant Quantum Computer (CRQC) would rende
...
...ical physics. Considerable research efforts and enormous corporate and government funding for the development of practical quantum computing systems are being invested currently. At the time of writing the document, Cryptographically Relevant Quantum Computers (CRQCs) that can break widely used public-key cryptographic algorithms are not yet available. However, it is worth noting that there is ongoing research and development in the field of quantum computing, with the goal of building more powerful and scalable quantum computers. One common myth is that quantum computers are faster than conventional CPUs and GPUs in all areas. This is not the case; much as GPUs outperform general-purpose CPUs only on specific types of problems, so too will quantum computers have a niche set of problems on which they excel; unfortunately for cryptographers, integer factorization and discrete logarithms, the mathematical problems underpinning all of modern cryptography, happen to fall within the niche that we expect quantum computers to excel at. As such, as quantum technology advances, there is the potential for future quantum computers to have a significant impact on current cryptographic systems.
```
Comment (tireddy2) [inline draft-ietf-pquip-pqc-engineers.md:181]: You can add the following line at the end of the paragraph: Predicting the emergence of CRQC is a challenging task, and there is ongoing uncertainty regarding when they will become practically feasible.

File: draft-ietf-pquip-pqc-engineers.md [hunk 2]
```diff
@@ -186,6 +186,8 @@ It is crucial for the reader to understand that when the word "PQC" is mentioned
...
 
+There is some discussion as to whether to use the term "Post-Quantum" or "Quantum Ready/Resistant" to denote algorithms that are resistant to CRQCs but no consensus has beenr reached. We would like to point out that Post-Quantum does not refer to algorithms that needs to be made quantum safe after the arrival of CRQCs but rather before that are based on hard cryptographically relevant mathematical problems that a CRQC as well as classical computers will not be able to break. On the other hand, quantum resistant/ready algorithms are such that have already been thought to be resistant to an attack by a CRQC but maybe vulnerable to traditional cryptanalysis and need not necessarily only consider algorithms based cryptographically hard math problems.
```
Comment (tireddy2) [inline draft-ietf-pquip-pqc-engineers.md:189]: I would rephrase the text as follows:
While there is ongoing discussion about whether to use the term 'Post-Quantum' or 'Quantum Ready/Resistant' to describe algorithms that resist CRQCs, a consensus has not yet been reached. It's important to clarify that 'Post-Quantum' refers to algorithms designed to withstand attacks by CRQCs and classical computers alike. These algorithms are based on mathematically hard cryptographic problems that neither CRQCs nor classical computers are expected to break.

On the other hand, 'quantum-resistant' or 'quantum-ready' algorithms have been designed to withstand CRQC attacks, but they may still be vulnerable to traditional cryptanalysis. Additionally, these algorithms may not exclusively rely on mathematically hard problems.

File: draft-ietf-pquip-pqc-engineers.md (hunk 1 above)
Comment (auriee) [reply draft-ietf-pquip-pqc-engineers.md:181]: Done.

File: draft-ietf-pquip-pqc-engineers.md (hunk 2 above)
Comment (auriee) [reply draft-ietf-pquip-pqc-engineers.md:189]: Done.

**Pull request merged**